### DISCUSSION TITLE
PCBs

### DISCUSSION THREAD

**Discussion by JKonopacki** [nebhead/PiFire#32] [General]
nebhead,

Really cool project.  I read you had extra PCBs from your build.  Any chance I can buy one?
Thanks
Jeff

**Comment by nebhead** [comment]
I have a couple more, would be happy to send you one.  Send me an e-mail with your mailing address at [redacted].

**Reply by weberbox** [reply]
I have five of the original PCB's that I ordered before the redesign I did. If interested I can send them over to you or if anyone needs one I can send them out  just let me know

**Reply by nebhead** [reply]
Hang on to them and if I run out, we can mail a few out to others that might need them.  Thanks for the offer though!

**Reply by dgshue** [reply]
I'm interested in one as well, if it's not too late! Excited about this project.

**Reply by nebhead** [reply]
@dgshue - Sure, just send me an e-mail at nebhead77 at gmail.com.
